src/Chapter02.Tests/Listing02.21.Tests.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Text.Json;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter02.Listing02_21.Tests;
+
+[TestClass]
+public class RawLiteralTests
+{
+    [TestMethod]
+    public void Main_OutputsJson()
+    {
+        string result = IntelliTect.TestTools.Console.ConsoleAssert.Execute("", Program.Main);
+
+        try
+        {
+            JsonDocument.Parse(result);
+
+        }
+        catch (JsonException ex)
```

Comment:
Do we just want to catch a JsonException or should we catch any exception?

Reply:
I would propose that we don't catch any exception and let the test fail since that is what it will do anyway